README.md

```diff
@@ -1,70 +1,43 @@
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+Frist Microverse projects to get start with React.
+The math magician web app is a simple calculator for solving minor math problems.
 
-### `npm run build`
+## Screenshoot
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+![Screeshot](./src/images/Screenshot.png)
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+## see the project
+   here is the:  [live_Demo]()
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+## Built With
 
-### `npm run eject`
+- React
+- SASS/SCSS
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+### Prerequisites
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+clone repo: `git clone https://github.com/Kandy-Peter/math-magicians.git`
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
-
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
-
-## Learn More
-
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
-
-To learn React, check out the [React documentation](https://reactjs.org/).
-
-### Code Splitting
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+# Getting Started with Create React App
 
-### Analyzing the Bundle Size
+run `npx create-react-app app_name` for a new repo.
+or `npx create-react-app .` for an existing repo.
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+### `npm start`
 
-### Making a Progressive Web App
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+The page will reload when you make changes.\
```

Comment:
- [ ] Please write a proper getting started section. It should be about how to get started with your own project and not the `Create React App`. This section should detail how one can get a local copy of your project, install dependencies and properly run the project locally on a computer.

- [ ] May you also ensure that your README has all the sections suggested in this [Microverse README template](https://github.com/microverseinc/readme-template) for students.